src/utils.py

```diff
@@ -55,3 +56,9 @@ def yield_gevent():
     # using a value of 0 means that this greenlet could be reschedualed again immediately so
     # we sleep for a small positive value to ensure another greenlet gets schedualed
     gevent.sleep(0.001)
+
+def geventwebsocket_handler_logger():
```

Comment:
we have a module called **log_utils.py**

I suggest in the `setup_logging`  function

```
if level < logging.WARNING:
    # geventwebsocket.handler logger pollutes our log with http requests at the INFO level
    # we raise the logging level for that module because we can see the http request in our
    # proxy server logs and want to make the app logs easier to parse
    logging.getLogger('geventwebsocket.handler').setLevel(logging.WARNING)
```

Reply:
added the code above.